templates/todo/projects/nodejs-mongo-swa-func/README.md

```diff
@@ -14,19 +14,9 @@ Let's jump in and get the ToDo app up and running in Azure. When you are finishe
 
 The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
 
-- [Azure Developer CLI](https://aka.ms/azure-dev/install)
-  - Windows:
-    ```powershell
-    powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
-    ```
-  - Linux/MacOS:
-    ```
-    curl -fsSL https://aka.ms/install-azd.sh | bash
-    ```
-- [Azure CLI (2.38.0+)](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [Azure Developer CLI](https://aka.ms/azd-install)
 - [Azure Functions Core Tools (4+)](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
 - [Node.js with npm (16.13.1+)](https://nodejs.org/) - for API backend and Web frontend
-- [Git (2.36.1+)](https://git-scm.com/)
```

Comment:
Why is git not a dependency any more?

Reply:
It's included in the install page on Dev Hub.